spec.md

```diff
@@ -1116,16 +1056,24 @@ message NodeServiceCapability {
 }
 ```
 
-### Error Type/Codes
+### Error Handling
+
+All CSI API calls defined in this spec MUST return a [standard gRPC status](https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto).
```

Comment:
Double check, but I think this is implied by the protocol on error.

Reply:
I don't see anywhere in the gRPC docs where it is prescribed, so I want it to be very explicit for CSI.

Reply:
https://github.com/grpc/grpc/blob/master/doc/statuscodes.md

> All RPCs started at a client return a status object composed of an integer code and a string message.

There might be something more normative, elsewhere.